project1/styles.css

```diff
@@ -0,0 +1,113 @@
+/* for my pullquote */
+
+blockquote {
+    margin: 0;
+}
+
+/* for my pullquote */
+
+blockquote p {
+    font-family: 'Satisfy', cursive;
+    font-size: 40px;
+    line-height: 1.25;
+    text-align: center;
+}
+
+/* for my pullquote */
+
+blockquote {
+    margin-left: 150px;
+    margin-right: 150px;
+}
+
+/* I wanted the background color of the <body> to be the same as the background color of the <main> element */
+
+body {
+    background-color: rgb(14, 190, 204);
+    color: whitesmoke;
+    margin: 125px;
+}
+
+
+/* for adding style to the bullets between my sections */
+
+div {
+    font-size: 18px;
+    letter-spacing: 6px;
+    line-height: 1;
+    text-align: center;
+}
+
+
+header {
+    color: navy;
```

Comment:
IMO, the navy doesn't mesh well with the blue background. Two things you can do here:

- Switch to using HSL colors and explore a darker version of the background color
- Put your background color in a color scheme generator like [coolors.co](https://coolors.co/0d9eaa-114b5f-e4fde1-456990-f45b69) and explore some ideas

Also consider if the third color you want to add here (you have the blue-green background and white already) should be a contrasting color (which may be one of the things that comes up via coolors.co) or just a darker shade of the background color. If the latter, does it make sense for the heading to be white (to tie in with the border and other header elements) and the body text to be a third color? This would help with readability, as the contrast between the white and blue at this font weight is okay, but not ideal.